### DISCUSSION TITLE
How can I use `=` to reindent _using vim_?

### DISCUSSION THREAD

**Discussion by wurli** [vscode-neovim/vscode-neovim#2301] [Q&A] [answered]
By default, `=` / `==` are mapped to `editor.action.formatSelection`, but I'd like to remove this mapping and instead let Neovim handle indentation for me.

I've tried adding a passthrough in my `keybindings.json` like so:
``` json
{
    "key": "=",
    "command": "vscode-neovim.send",
    "when": "editorTextFocus",
    "args": "=",
}
```

As per the advice in the README, I've looked in `keybindings.json` and `settings.json` to see if I can remove the default keybinding, to no avail. I've also tried setting `=` as a passthrough using `vscode-neovim.ctrlKeysForNormalMode`, but this doesn't seem to be a supported option.

Grateful for any advice!

**Comment by justinmk** [comment, marked as answer]
vscode-neovim's default mapping for "=" is defined in Nvim instead of vscode (not sure why). To unmap vscode-neovim's default for "=", just do this:

    :unmap =
    :unmap ==

**Reply by wurli** [reply]
Thank you! This was super quick and exactly what I needed 💯
